examples/research_projects/jax-projects/README.md

```diff
@@ -975,11 +975,44 @@ For more information, check out [this PR](https://github.com/huggingface/hugging
 
 ## How to setup TPU VM
 
-TODO (should be filled by 2.07.)...
+In this section we will explain how you can ssh into a TPU VM that has been given to you.
+If your username is in one of the officially defined projects [here](https://docs.google.com/spreadsheets/d/1GpHebL7qrwJOc9olTpIPgjf8vOS0jNb6zR_B8x_Jtik/edit?usp=sharing), you should have received two emails: 
 
-## How to use the hub for training and demo
- 
-TODO (should be filled by 1.07.)...
+- one that states that you have been granted the role "Community Week Participants" for the project hf-flax
+- one (or more if you are in multiple projects) that gives you your TPU name and your TPU zone.
+
+You should click on "Open Cloud Console" on the first mail and agree to the pop up windows that follows. It will allow you to use a TPU VM. Don't worry if you cannot access the actual project `hf-flax` visually on the google cloud console - that is expected!
```

Comment:
```suggestion
You should click on "Open Cloud Console" on the first mail and agree to the pop up windows that follows. It will allow you to use a TPU VM. Don't worry if you cannot access the actual project `hf-flax` visually on the google cloud console - this is expected!
```